draft-sharabayko-mops-srt.md

```diff
@@ -2086,15 +2086,583 @@ and RTTVar are updated by the socket based on algorithms for the sender
 When an SRT socket receives data, it updates its local RTT and RTTVar,
 which can be used for its own sender as well.
 
-## Congestion Control
+# SRT Packet Pacing and Congestion Control
+
+SRT provides certain mechanisms for the sender to get a feedback
+from the receiving side through acknowledgement (ACK) packets
+({{ctrl-pkt-ack}}). Every 10 milliseconds the sender receives
+the latest values of RTT, RTT variance, available buffer size,
+receiving rate and estimated link capacity. Upon reception of
+an NAK packet ({{ctrl-pkt-nak}}) the sender detects packet loss
+during the transmission. These mechanisms provide a solid background
+for various congestion control algorithms integration in the
+SRT protocol.
+
+Given that SRT can operate in live and file transfer modes, there are
+two groups of congestion control algorithms defined in SRT: live
+congestion control (LiveCC) and file transfer congestion control
+(FileCC).
```

Comment:
```suggestion
There are two groups of congestion control algorithms defined in SRT, 
corresponding to the two modes of operation: live congestion control (LiveCC), 
and file transfer congestion control (FileCC).
```

Reply:
As SRT is designed both for live streaming and file/message transmission {{link}}, there are two groups of congestion control algorithms defined in SRT: live congestion control (LiveCC), and file transfer congestion control (FileCC).

Reply:
we have two modes in SRT: message and buffer, it's not live and file transfer as I've figured out. Live streaming is possible with using message mode of SRT, file transfer - both message and buffer. We will be reworking [Data Transmission Modes](https://tools.ietf.org/html/draft-sharabayko-mops-srt-01#page-27) section a bit.

Reply:
Yes, your change makes it better.